js_modules/dagit/packages/app/package.json

```diff
@@ -39,7 +39,7 @@
     "eslint-webpack-plugin": "3.1.1",
     "prettier": "2.2.1",
     "typescript": "5.0.2",
-    "webpack": "^5.0.0",
+    "webpack": "^5.88.1",
```

Comment:
I upgraded Webpack from 5.81 as well while debugging. I can roll this back if you feel strongly.